### PR TITLE
Fix for MetisTuner

### PR DESCRIPTION
**Pull request by leelaylay**
1. Remove the debug information which can cause error.
2. Increase 'alpha' parameter of GaussianProcessRegressor to make sure a positive definite matrix.
3. Increase the probability of exploration from 0.1 to 0.9 to select from next config.

### CODE REVIEW

File: src/sdk/pynni/nni/metis_tuner/metis_tuner.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ class MetisTuner(Tuner):
     """
 
     def __init__(self, optimize_mode="maximize", no_resampling=True, no_candidates=True,
-                 selection_num_starting_points=600, cold_start_num=10, exploration_probability=0.1):
+                 selection_num_starting_points=600, cold_start_num=10, exploration_probability=0.9):
```
Comment (QuanluZhang) [inline src/sdk/pynni/nni/metis_tuner/metis_tuner.py]: why changing from 0.1 to 0.9, seems like a big change.

Comment (leelaylay) [reply src/sdk/pynni/nni/metis_tuner/metis_tuner.py]: I talked with @xuehui1991 about it before. Metis Tuner will do more exploration with next_config.

Comment (QuanluZhang) [reply src/sdk/pynni/nni/metis_tuner/metis_tuner.py]: is it better to expose this configuration to users? or you think 0.9 is a good choice for different applications.

Comment (leelaylay) [reply src/sdk/pynni/nni/metis_tuner/metis_tuner.py]: Sure, I do think 0.9 is just a good choice.

**Pull request merged**